### ISSUE TITLE
Power BI and xampp

### ISSUE THREAD

**Issue by arijmachfar** [Microsoft/PowerBI-visuals#414]
Hi,I'm working with xampp and I want to import my database to powerBI,so I had installed power bi desktop but when I'm trying to import my database Ihad this error.
Détails : « MySQL : Désolé... Nous n'avons pas pu trouver un fournisseur de base de données portant le nom invariant « MySql.Data.MySqlClient ».
help please!!

**Comment by ignatvilesov** [comment]
@arijmachfar Did you install [`mysql-connector-net-6.6.5.msi`](http://dev.mysql.com/downloads/connector/net/6.6.html) ?

**Comment by arijmachfar** [comment]
yes I do but it doesn't work I had this error "Détails : « MySQL : Unable to connect to any of the specified MySQL hosts. »"
